app/assets/stylesheets/styles/components/guides/_edit_guides.css.scss

```diff
@@ -108,30 +108,30 @@
       }
     }
 
-    .title{
+    .title {
       margin: 0;
       font-size: .9rem;
       margin-top: .5rem;
     }
 
-    &:hover{
+    &:hover {
       opacity: .7;
     }
 
-    input[type=range]{
+    input[type=range] {
       width: 66%;
       margin: .7rem 1rem 0 0;
     }
 
-    input[type=checkbox]{
+    input[type=checkbox] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3